hair_salon/data/ir_ui_view.xml

```diff
@@ -0,0 +1,70 @@
+            </gantt>
+        </field>
+        <field name="active" eval="True"/>
+        <field name="inherit_id" ref="appointment.calendar_event_view_gantt_booking_resource"/>
+        <field name="model">calendar.event</field>
+        <field name="name">calendar.event.view.gantt.meeting.user</field>
+        <field name="type">gantt</field>
+    </record>
+    <record id="appointment_type_customization" model="ir.ui.view">
+        <field name="arch" type="xml">
+                <xpath expr="//form[1]/sheet[1]/group[1]/group[@name='left_details']/div[1]" position="after">
+                    <field name="x_break" widget="boolean"/>
+                    <field name="x_break_start_appointment"  widget="float_time" invisible="not x_break"/>
+                    <field name="x_break_end_appointment"  widget="float_time" invisible="not x_break"/>
+                </xpath>
+        </field>
+        <field name="active" eval="True"/>
+        <field name="inherit_id" ref="appointment.appointment_type_view_form"/>
+        <field name="mode">extension</field>
+        <field name="model">appointment.type</field>
+        <field name="name">appointment.type.form customization</field>
+        <field name="priority">200</field>
+        <field name="type">form</field>
+    </record>
+    <odoo>
+        <record id="calendar_event_view_form_gantt_booking_inherit" model="ir.ui.view">
+            <field name="name">calendar.event.view.form.gantt.booking.inherit</field>
+            <field name="model">calendar.event</field>
+            <field name="inherit_id" ref="appointment.calendar_event_view_form_gantt_booking"/>
+            <field name="active" eval="True"/>
+            <field name="arch" type="xml">
+                    <xpath expr="//form/sheet/group/group" position="after">
+                        <div class="o_form_group o_inline" invisible="not x_parent_id">
+                            <label for="x_parent_id" string="Linked to:" class="oe_inline me-2"/>
+                            <field name="x_parent_id" options="{'no_create': True}" class="oe_inline"/>
+                        </div>
+                    </xpath>
+            </field>
+        </record>
+    </odoo>    
+    <record id="calendar_event_parent_customization" model="ir.ui.view">
+        <field name="arch" type="xml">
+                <xpath expr="//form[1]/sheet[1]/group[1]/group[1]/div[1]" position="after">
+                    <field name="x_break_start" invisible="x_parent_id" widget="float_time"/>
+                    <field name="x_break_end" invisible="x_parent_id" widget="float_time"/>
+                </xpath>
+                <xpath expr="//field[@name='location']" position="before">
+                    <field name="x_appointment_part_2" string="Part 2" invisible="not x_appointment_part_2"/>
+                    <field name="x_parent_id" invisible="not x_parent_id"/>
+                </xpath>
```

Comment:
What about part_1?

Reply:
We have to chose either how to avoid cascade effect?